projects/netgrif-components-core/src/assets/i18n/en.json

```diff
@@ -456,5 +462,16 @@
     },
     "dynamicNavigation": {
         "couldNotResolveView": "An error has occurred during the view resolution process. The view could not be displayed"
+    },
+    "impersonation": {
+        "user": {
+            "successfullyRepresented": "User successfully represented",
+            "currentlyAlreadyRepresented": "User is already currently represented",
```

Comment:
"already currently" does not sound right to me.

I would try "User is already being represented by someone else".

Reply:
done